dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -78,6 +85,13 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             THROW_IF_FAILED(notificationPlatform->AddToastRegistrationMapping(processName.get(), toastAppId.c_str()));
         }
 
+        if (AppModel::Identity::IsPackagedProcess())
```

Comment:
REcommend: Create a new Register() API in the idl and reimplement this function in that. You don't have to delete this until that implementation is complete.

Reply:
Apologies I have already Modified the existing API and got rid of the arguments. Hopefully it is perfect :)